release-notes/5.0/5.0-supported-os.md

```diff
@@ -89,8 +85,6 @@ The following operating systems are no longer supported, starting with .NET 5.0.
 
 Support for the following versions was ended by the distribution owners and are [no longer supported by .NET 5][OS-lifecycle-policy].
 
-|OS         | Version  | End of Life | Supported Version|
-|-----------|----------|-------------|------------------|
-| Windows 10| 1703     | [10/08/2019](https://support.microsoft.com/help/13853/windows-lifecycle-fact-sheet) |
+None yet.
```

Comment:
Above it says 'Arm32' - could you uppercase ARM everywhere?

Pedantically, I assume it's "Windows for ARM" since I don't think it's officially ARM32. But we should at least upper case everywhere.

Reply:
It is the opposite. It should be "Arm" everywhere. That's the correct casing.

Reply:
Interesting - you're right! TIL. Oddly enough on https://www.arm.com/ it's all lower case when they refer to the compay.

Reply:
Yes, the company is "arm" and the architecture is "Arm", or at least that is my understanding. The arm.com website isn't entirely consistent, but I think that is due to the legacy of the previous all-capital plan that you were earlier referring to. I switched the casing on the these docs late last year. You were not the first to raise this issue.